crates/sui-framework/tests/coin_tests.move

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+#[test_only]
+module sui::test_coin {
```

Comment:
nit should be `coin_tests`?

Reply:
am a fan of consistent naming/conventions, appreciate!